Allow forcing delta lake to recalculate all statistics (#16634)
trinodb/trino: docs/src/main/sphinx/connector/delta-lake.rst

  ANALYZE table_schema.table_name;

To recalculate from scratch the statistics for the table use additional parameter ``full_refresh``.

>   ANALYZE table_schema.table_name WITH(full_refresh = true);

Reviewer: How about changing to enum (`mode` or something) instead of boolean? Are we sure that we don't add new mode in the future?

  ANALYZE table_schema.table_name;

> To recalculate all stats for the table use additional parameter ``full_refresh``.

Reviewer 2: ```suggestion
To recalculate from scratch the statistics for the table, use additional parameter ``full_refresh``.
```

Can be a follow-up

  ANALYZE table_schema.table_name;

> Because Delta Lake connector can also update stats on writes they can drift away from

Reviewer 3: Drifting from proper values are because we don't update statistics in some situations not the opposite.

> Because Delta Lake connector can also update stats on writes they can drift away from

Author: I will follow Marius advise and delete explanation

  ANALYZE table_schema.table_name;

> To recalculate from scratch the statistics for the table use additional parameter ``mode``.

Reviewer: Please replace the `.` with `:`. Otherwise, the highlight doesn't work in the following SQL.

  ANALYZE table_schema.table_name WITH(mode = 'full_refresh');

```mode``` can have 2 different values: ```full_refresh``` or ```incremental```.

> The latter one is a default one.

Reviewer: How about:
```
There are two modes available ``full_refresh`` and ``incremental``.
The procedure use ``incremental`` by default.
```
